include/eve/detail/function/simd/common/make.hpp

```diff
@@ -36,20 +36,34 @@ namespace eve::detail
     }(std::make_index_sequence<s_t{}.size()>{});
   }
 
-  template<typename T, typename N, typename... Vs>
+  template<arithmetic_scalar_value T, typename N, typename... Vs>
   EVE_FORCEINLINE auto make(eve::as<wide<T, N>> const &, Vs... vs) noexcept
     requires std::same_as<abi_t<T, N>, emulated_>
   {
     return make_emulated<wide<T, N>>(vs...);
   }
 
-  template<typename T, typename N, typename... Vs>
+  template<has_plain_translation T, typename N, typename... Vs>
+  EVE_FORCEINLINE auto make(eve::as<wide<T, N>> const &, Vs... vs) noexcept
+    requires std::same_as<abi_t<T, N>, emulated_>
+  {
+    return bit_cast(make(eve::as<wide<translate_t<T>, N>>{}, static_cast<translate_t<Vs>>(vs)...), as<wide<T, N>>{});
+  }
+
+  template<arithmetic_scalar_value T, typename N, typename... Vs>
   EVE_FORCEINLINE auto make(eve::as<logical<wide<T, N>>> const &, Vs... vs) noexcept
     requires std::same_as<abi_t<T, N>, emulated_>
   {
     return make_emulated<logical<wide<T, N>>>(vs...);
   }
 
+  template<has_plain_translation T, typename N, typename... Vs>
+  EVE_FORCEINLINE auto make(eve::as<logical<wide<T, N>>> const &, Vs... vs) noexcept
+    requires std::same_as<abi_t<T, N>, emulated_>
+  {
+    return bit_cast(make(eve::as<logical<wide<translate_t<T>, N>>>{}, static_cast<translate_t<Vs>>(vs)...), as<logical<wide<T, N>>>{});
+  }
```

Comment:
@jfalcou  - how much is this in the same boat as operators.

I think it might be beneficial if it was, so that we don't compile this for non-native types.

Reply:
As stated earlier, we are going to migrate everything to callable as much as it makes sense.